tests/testthat/test-krcomp.R

```diff
@@ -71,3 +71,50 @@ test_that("kr give similar results as SAS", {
   expect_equal(res$df, expected[2], tolerance = 1e-3)
   expect_equal(res$se, expected[1], tolerance = 1e-3)
 })
+
+# h_df_1d_kr
```

Comment:
```suggestion
# h_df_1d_kr ---
```
and elsewhere as well. see conventions in other files. this helps with navigating large tests files. thanks!